### PR TITLE
bump karma & angular cli

### PR DESCRIPTION
**Pull request by lauraGgit**
doesn't fix the auth component test failure test, but  bumps up a bunch of dependencies and config.

### CODE REVIEW

File: frontend/src/app/authentication/authenticated.component.ts [hunk 1]
```diff
@@ -48,6 +48,9 @@ export class AuthenticatedComponent implements OnInit {
     localStorage.removeItem('token');
     this.user = null;
 
+    console.log(this.authentication);
+    console.log(this.authentication.removeUser());
```
Comment (hursey013) [inline frontend/src/app/authentication/authenticated.component.ts]: These can be removed

**Comment by lauraGgit** [comment]
@hursey013 updated to remove the console logs

File: frontend/package.json [hunk 2]
```diff
@@ -19,8 +19,8 @@
     "build-test-pa11y": "./development-configurations/build-test-pa11y.sh",
     "snyk-test": "snyk test",
     "start": "ng serve",
-    "test:ci": "ng test --sourcemaps=false --config ./development-configurations/karma.conf --single-run=true --watch=false --progress=false --code-coverage",
-    "test": "ng test --config ./development-configurations/karma.conf --sourcemaps=true",
+    "test:ci": "ng test --karma-config ./development-configurations/karma.conf --single-run=true --watch=false --progress=false --code-coverage",
```
Comment (hursey013) [inline frontend/package.json]: It looks like these changes may have broken the e2e tests on circleci, they don't appear to run in the associated test https://circleci.com/gh/18F/fs-permit-platform/1019?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: frontend/package.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
     "dist-staging": "ng build --prod --env=staging --aot=false",
     "dist-prod": "ng build --prod --env=prod --aot=false",
     "docs": "yarn run typedoc --out ./src/assets/typedoc --exclude '**/*.spec.ts' ./src/ --module commonjs",
-    "e2e:ci": "ng e2e --port=4200 --watch=false --progress=false",
-    "e2e:cidkr": "ng e2e --port=4200 --watch=false --progress=false --config ./development-configurations/protractor.conf.js --specs=./e2e/authenticated/special-uses/application-noncommercial-group.e2e-spec.ts",
+    "e2e:ci": "ng e2e --port=4200",
```
Comment (hursey013) [inline frontend/package.json]: Is `--progress=false` still a valid flag? Would adding that back in remove the compiling output from CircleCI?

Comment (hursey013) [reply frontend/package.json]: Scratch that, I was getting this confused with the frontend unit test.

Comment (lauraGgit) [reply frontend/package.json]: Looking at the cli docs, I don't think it is. https://github.com/angular/angular-cli/wiki/e2e but i can check locally

**Comment by hursey013** [comment]
Looks like the `unauthenticated e2e tests` are not being run due to the `--config` flag

**Comment by lauraGgit** [comment]
@hursey013 I recommend that we pull the `--progress=false` that's a now deprecated flag that would end the tests rather than hide the massive bundling. Sadly, I'm not sure that we can hide that now without some customization.

**Pull request merged**